mmv1/templates/terraform/constants/redis_version.go

```diff
@@ -0,0 +1,25 @@
+// Is the new redis version less than the old one?
+func isRedisVersionDecreasing(_ context.Context, old, new, _ interface{}) bool {
```

Comment:
Do we need this? What is the failure mode for when a user attempts to downgrade the redis version via the API?

Can you add a test for downgrading the version?

Reply:
you can only upgrade the version. api fails if you try and downgrade

```
Error: Error updating Instance "projects/scottsuarez-graphite/locations/us-central1/instances/booby": googleapi: Error 400: Upgrade to Redis version REDIS_4_0 is not supported. The following versions are available: REDIS_6_X.
com.google.apps.framework.request.StatusException: <eye3 title='INVALID_ARGUMENT'/> generic::INVALID_ARGUMENT: Upgrade to Redis version REDIS_4_0 is not supported. The following versions are available: REDIS_6_X.
Details:
[
  {
    "@type": "type.googleapis.com/google.rpc.BadRequest",
    "fieldViolations": [
      {
        "description": "Invalid value: REDIS_4_0",
        "field": "redis_version"
      }
    ]
  }
]
```

Reply:
yeee